### DISCUSSION TITLE
packer: Build arm64_big_sur bottle

### DISCUSSION THREAD

**Discussion by Tenzer** [Homebrew/discussions#1519] [Everyday usage] [answered]
Hey.

The issue reporting template didn't really fit with this, which is why I'm trying here instead.

Packer 1.7.2 added support for being built natively for macOS ARM64: https://github.com/hashicorp/packer/issues/10708#issuecomment-824166157, but Homebrew isn't currently building and publishing bottles for it.

I had a look at the formula for Packer and didn't find any obvious way to enable it. Looking at how bottles were enabled for Terraform, it seemed to have been done manually: https://github.com/Homebrew/homebrew-core/commit/3b2251236806df6b4a23957ff95ead56d7051270.

I have verified that `brew install --build-bottle packer` works correctly, running natively on an ARM64 Mac, so I think it should work fine if we could just get the bottle building enabled for this architecture.

**Comment by carlocab** [comment, marked as answer]
Bottled in https://github.com/Homebrew/homebrew-core/commit/cde3eeb98d704de02f21753ea0a60c42111a4f97.

**Reply by Tenzer** [reply]
Thank you! So, would this be the best way to get bottles enabled for Darwin ARM64, or is there any other way I could have done it?

**Reply by carlocab** [reply]
If you're able to run `brew install -s foo && brew test foo && brew audit --strict foo` without any errors for some formula `foo` that isn't yet bottled for ARM, then sure, please do let us know here.

**Reply by Tenzer** [reply]
Alright, will keep that in mind. For now, I've just been checking every once in a while if any of the packages I had installed via Rosetta 2 were available to run natively. The last two formulas I've got are both Haskell ones and they don't seem to be available natively on ARM64 yet, due to `ghc` not yet being available ([work is in underway](https://gitlab.haskell.org/ghc/ghc/-/issues/18664)) so they will have to wait.
